src/plugins/README.md

```diff
@@ -101,6 +101,7 @@ Only suited for import/export:
 - [xmltool](xmltool/) uses XML (in Elektra's XML schema).
 - [simpleini](simpleini/) line-based key-value pairs with configurable
   format (without sections)
+- [mini](mini/) minimalistic dependency free plugin for the INI  file format.
```

Comment:
> But shouldn't we call such plugins `properties` (as in https://en.wikipedia.org/wiki/.properties) instead?

That would make sense, although I suspect that more people know about the INI format, than about `.properties` files. The syntax of `.properties` looks a bit different compared to the standard INI format too. As far as I can tell, `mini` does not support any of these two formats properly.

> Or do you have in mind to add sections later?

No.